packages/komencikit/src/login.ts

```diff
@@ -0,0 +1,29 @@
+import { Address } from '@celo/contractkit'
+import { EIP712TypedData } from '@celo/utils/lib/sign-typed-data-utils'
+
+export const buildLoginTypedData = (
```

Comment:
Yeah we can add them, not necessarily to the login struct because I think having the captcha token is enough for replay protection. But currently I stripped these from the client under the impression that we'll stick only with captcha. If we feel confident in adding those I can do that but I'd merge this as is in the mean time to have a starting point for the mobile work.

Reply:
Sounds good. Feel free to merge as is and we can always add those in once we have more clarity.